resources/lib/sutils.py

```diff
@@ -170,8 +179,8 @@ def add_item_to_library(self, item_path, item_url):
         error = False
         new = False
         if item_path:
-            item_path = xbmc.translatePath( item_path )
-            dir = os.path.dirname(item_path)
+            item_path = os.path.normpath(xbmc.translatePath( item_path ))
```

Comment:
k cemu je doby os.path.normpath? v commitu 1153588 jsem to odstranil, aby fungovalo ukladani i na specialni cesty